corehq/form_processor/backends/sql/dbaccessors.py

```diff
@@ -867,14 +867,15 @@ def get_indexed_case_ids(domain, case_ids):
             return [result.referenced_id for result in results]
 
     @staticmethod
-    def get_reverse_indexed_cases(domain, case_ids):
+    def get_reverse_indexed_cases(domain, case_ids, case_types=None, is_closed=None):
         assert isinstance(case_ids, list)
+        assert case_types is None or isinstance(case_types, list)
         if not case_ids:
             return []
 
         cases = list(CommCareCaseSQL.objects.raw(
-            'SELECT * FROM get_reverse_indexed_cases(%s, %s)',
-            [domain, case_ids])
+            'SELECT * FROM get_reverse_indexed_cases_3(%s, %s, %s, %s)',
```

Comment:
Why `_3` instead of `2`?

Reply:
There's the sql function name, the name of the `.sql` file containing the function, and the analog of both of those in `sql_proxy_accessors`.  I figured it'd be a little saner to use the same number for all four of those (the file name needed to be 3, since the last version was 2).